docs/application/web/api/4.0/ui_fw_api/Wearable_UIComponents/wearable_component_list.htm

```diff
@@ -42,6 +42,10 @@ <h1>Wearable UI Components</h1>
 				<td><a href="wearable_circularindexscrollbar.htm">Circular index scroll bar</a></td>
 				<td>Shows a circular index scroll bar which uses rotary.</td>
 			</tr>
+			<tr>
```

Comment:
I can see there are 3 added pages in 4.0.

Would you please check the application/web/api/toc.xml file also and add these 3 files to the related location, under 4.0?

Reply:
done